console/account/troubleshooting/cannot-erase-account.mdx

```diff
@@ -0,0 +1,40 @@
+---
+meta:
+  title: I cannot erase my account and personal data
+  description: This page helps you permanently erase your account
+content:
+  h1: I cannot erase my account and personal data
+  paragraph: This page helps you permanently erase your account
+tags: account close delete deactivate close-account
+dates:
+  validation: 2024-03-28
+  posted: 2024-03-28
+categories:
+  - console
+---
+
+## My request to erase my account and personal data has failed
+
+### Problem
+When you click the **Send request** button, an error message appears. This message says that you still have active resources.
+
+### Cause
+You have active resources, preventing us from proceeding with your request. 
+
+### Solution 
+You can check all your active resources from your **Organization dashboard**. If one or more resources are listed, they are active. You need to delete all of them manually. In this situation, you must wait for the current month's invoice to be paid before sending another request to erase your account. 
```

Comment:
Should we also advise to check in the Organization's Projects to make sure there are no resources left in other Projects (than the default one)?